code/_helpers/_scp.dm

```diff
@@ -0,0 +1,7 @@
+//SCP Helpers
+
+///Checks if atom is an SCP
+/proc/isSCP(atom/A)
```

Comment:
This seems a bit redundant, no?
Would make sense if it was an atom proc, so you could override results on case-by-case basis, but as it is right now - it is rather pointless and you can simply check if(A.SCP) instead.

Reply:
Yes, this was when I just started, I just ported all existing stuff and decided to wait until I do the final cleanup and pass over stuff to see if it's necessary. Probably will remove it before I mark this as ready for review.